async port scaner.py

```diff
@@ -0,0 +1,31 @@
+import socket
+import asyncio
+import time
+t=time.time()
+port=0
+async def scan(i: int):
+    """
+    check if a port for a socket is valid and save it as global i if True
```

Comment:
If you describe the function `scan` with `"check"` then rename the function appropriately to more indicative name; i.e. `check_port`.